### DISCUSSION TITLE
Add light white hover color back for login prompt placeholders

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#30378] [Ideas] [answered]
For sign-in placeholders, it's quite difficult to find it clickable, since there is only a slight sound when the cursor hovers on them.

After some research I've found that the hover background content is overwritten in the definition of placeholders, and I managed to add them back, see the pictures below.

![Current version of placeholder](https://github.com/user-attachments/assets/9ac55189-4574-4366-819a-93a4d672a1c5)
![Modified version of placeholder](https://github.com/user-attachments/assets/d54ce12a-99d4-4ed1-a178-f3cc74af0918)

Should we add it back?

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30383
